### ISSUE TITLE
'find' command can not search files under sails/sails-disk directories

### ISSUE THREAD

**Issue by davidxiao** [balderdashy/sails#3004]
Hi, 
   I tried to run 'find' some files in project root directory like test-sails, 'find' automatically ignores the sails/sails-disk under node_modules, any ideas about how to enable it? but could find files under non-sails directories

For example:  
vagrant$ pwd
/vagrant/test-sails
vagrant$ find . -name '*.js' | grep sails # There must be many js files under ./node_modules/sails
./.tmp/public/js/dependencies/sails.io.js
./assets/js/dependencies/sails.io.js
./tasks/config/sails-linker.js

**Comment by davidxiao** [comment]
$ sails new test-sails
$cd test-sails
$find . -name '*.js' | grep sails
./assets/js/dependencies/sails.io.js
./tasks/config/sails-linker.js
# while the below command could seach many files.
$find ./node_modules/sails/node_modules/ -name '*.js' | grep sails | more

**Comment by sailsbot** [comment]
Thanks for posting, @davidxiao.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
